integration-cli/docker_cli_stats_test.go

```diff
@@ -39,5 +39,20 @@ func (s *DockerSuite) TestCliStatsNoStream(c *check.C) {
 		statsCmd.Process.Kill()
 		c.Fatalf("stats did not return immediately when not streaming")
 	}
+}
+
+func (s *DockerSuite) TestStatsContainerNotFound(c *check.C) {
+	testRequires(c, DaemonIsLinux)
```

Comment:
I've turned off almost all CI tests when running against a Windows daemon. Systematically, we're starting turning them on as modifications are done eg #16312